app.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import argparse
+
+
+import settings
+from easy_phi_client import PythonClient
+
+# parse parameters
+parser = argparse.ArgumentParser(
+description = "Send API calls to easy_phi web application. "
+			  "Results will be printed to console")
+parser.add_argument('-s', '--slot', default=0, help='slot number, int')
+parser.add_argument('-scpi', '--scpi_command', help='scpi command, char')
+args = parser.parse_args()
+
+slot = args.slot
+scpi = args.scpi_command
+pc = PythonClient(settings.api_token)
+
+# platform info
```

Comment:
not sure why you do it all. You can accept a choice parameter to check what user actually wants to do, or just send scpi commands, or demonstrate simple scenario. Also, api_token should not be stored in settings